### PR TITLE
Add method to get operations from features

### PR DESCRIPTION
**Pull request by charithag**

### CODE REVIEW

File: features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/modules/operation.js [hunk 1]
```diff
@@ -27,7 +27,42 @@ var operationModule = function () {
...
+            for (var i = 0; i < features.size(); i++) {
+                feature = {};
+                feature["id"] = features.get(i).getId();
+                feature["code"] = features.get(i).getCode();
+                feature["name"] = features.get(i).getName();
+                feature["description"] = features.get(i).getDescription();
+                feature["deviceType"] = features.get(i).getDeviceType();
+                feature["metadataEntries"] = [];
+                var metaData = features.get(i).getMetadataEntries();
+                if (metaData && metaData != null) {
```
Comment (harshanL) [inline features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/modules/operation.js:46]: Having if (metaData) is enough here.

File: features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/modules/operation.js [hunk 2]
```diff
@@ -27,7 +27,42 @@ var operationModule = function () {
...
+                    for (var j = 0; j < metaData.size(); j++) {
+                        metaDataEntry = {};
+                        metaDataEntry["id"] = metaData.get(j).getId();
+                        metaDataEntry["value"] = metaData.get(j).getValue();
+                        ["metadataEntries"].push(metaDataEntry);
+                    }
+                }
+                featureList.push(feature);
+            }
+            log.info(featureList);
```
Comment (harshanL) [inline features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/modules/operation.js:57]: Remove the unnecessary log here.

Comment (charithag) [inline features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/modules/operation.js:57]: Fixed in commit https://github.com/wso2/carbon-device-mgt/commit/e537e2d841ca3c7946cd34e9da96f261e0ac94e1

File: features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/modules/operation.js (hunk 1 above)
Comment (charithag) [inline features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/modules/operation.js:46]: Fixed in commit https://github.com/wso2/carbon-device-mgt/commit/e537e2d841ca3c7946cd34e9da96f261e0ac94e1

**Pull request merged**